src/modules/home/components/carousel.tsx

```diff
@@ -0,0 +1,35 @@
+import { styled } from 'nativewind';
+import React from 'react';
+import { FlatList as RNFlatList } from 'react-native';
+
+import type { Product } from '../../../api/types';
+import { common } from '../../../translations/en.json';
+import { Text } from '../../../ui/core/text';
+import { View } from '../../../ui/core/view';
+import { CarouselItem } from './carousel-item';
+const FlatList = styled(RNFlatList);
+
+type Props = {
+  data: Product[];
+  lenght: number;
+};
+
+export const HorizontalCarousel = ({ data }: Props) => {
+  return (
+    <>
+      <FlatList
+        className="h-[300]"
+        data={data}
+        contentContainerStyle={{ alignItems: 'center', gap: 16, paddingHorizontal: 16 }}
```

Comment:
in line styles ;)

Reply:
I analyze the option of using Tailwind React Native Classnames library to style in the contentContainerStyle.